quickwit/quickwit-integration-tests/src/tests/ingest_v2_tests.rs

```diff
@@ -332,6 +332,83 @@ async fn test_ingest_v2_happy_path() {
     sandbox.shutdown().await.unwrap();
 }
 
+#[tokio::test]
+async fn test_ingest_v2_high_throughput() {
+    let sandbox = ClusterSandboxBuilder::build_and_start_standalone().await;
+    let index_id = "test_high_throughput";
+    let index_config = format!(
+        r#"
+        version: 0.8
+        index_id: {index_id}
+        doc_mapping:
+            field_mappings:
+            - name: body
+              type: text
+        indexing_settings:
+            commit_timeout_secs: 1
+        "#
+    );
+    sandbox
+        .rest_client(QuickwitService::Indexer)
+        .indexes()
+        .create(index_config, ConfigFormat::Yaml, false)
+        .await
+        .unwrap();
+
+    let body_size = 20 * 1000 * 1000;
+    let line = json!({"body": "my dummy repeated payload"}).to_string();
+    let num_docs = body_size / line.len();
+    let body = std::iter::repeat_n(&line, num_docs).join("\n");
+    let ingest_resp = sandbox
+        .rest_client(QuickwitService::Indexer)
+        .ingest(
+            index_id,
+            IngestSource::Str(body),
+            // TODO: when using the default 10MiB batch size, we get persist
+            // timeouts with code 500 on some lower performance machines (e.g.
+            // Github runners). We should investigate why this happens exactly.
+            Some(5_000_000),
```

Comment:
@guilload I didn't find a good explanation for why this timeout occur here in the persist https://github.com/quickwit-oss/quickwit/blob/ce4501fc9cde184273a17a09cac1bc6cf55173b4/quickwit/quickwit-ingest/src/ingest_v2/router.rs#L432-L443

Persisting 10MB should not take 6 sec, even on a slow system, should it?